check_pgactivity

```diff
@@ -3487,6 +3490,95 @@ sub check_database_size {
 }
 
 
+=item B<extensions_versions> (9.1+)
+
+Check all extensions installed in all databases
+and raise an alarm if the current version is not the default version available
+on the instance.
+
+Typically, it is used to detect forgotten extension upgrades after package
+upgrades or a pg_upgrade.
+
+Perfdata returns the number of outdated extensions in each database.
+
+The 'postgres' database and templates are always excluded.
```

Comment:
I don't think we should exclude any database by default.  It's especially important for template databases if people relies on the template databases to be up to date to create new production databases.

Reply:
I agree.
I've just used get_all_dbname, like many other services. Perhaps it should get a parameter to avoid such filtering.

Reply:
Solved by PR #224